quiz/mobile/__tests__/App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import renderer from "react-test-renderer";
+
+import App from "../App";
```

Comment:
I'd suggest setting up absolute imports, to solve problem of `../../../../../../`